### PR TITLE
 Add functionalities

### PR DESCRIPTION
**Pull request by koachcode**
- Create forms to perform the following functions:
  - Creates a Post on behalf of the current_user.
  - Create a comment on behalf of the current_user.
- Allow Users to add likes to Posts.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,13 @@
 Rails.application.routes.draw do
   resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
+    resources :posts, only: %i[index show new create] do
+      resources :likes, only: %i[new create]
+      resources :comments, only: %i[new create]
+    end
   end
```
Comment (DeliceLydia) [inline config/routes.rb]: - [ ] Please work on these routes ` POST   /posts/:post_id/comments`
 `GET   /posts/:post_id/comments/new` means you should have a route that takes you to the page where you create a new comment, and another one that shows post comments currently they are not working

Comment (DeliceLydia) [reply config/routes.rb]: - [ ] Remember to work on this route `POST /posts/:post_id/likes` for creating a like on a post

**Review by DeliceLydia — APPROVED** [review]
## Hi @Ginohmk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Kudos on:
- Writing a good README
- Following git workflow
- Setting up correctly linters

## Required Changes ♻️
_Check the comments under the review._



### Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: config/routes.rb (hunk 1 above)
Comment (koachcode) [reply config/routes.rb]: Hello Like are created and for the comments there are created too

Comment (koachcode) [reply config/routes.rb]: A Video by Kanu - May 4th 2022 11_13 pm

Comment (koachcode) [reply config/routes.rb]: PLease take a look at the video and see the comment and the likes working\ @DeliceLydia

Comment (koachcode) [reply config/routes.rb]: I deliberately did not create my comment on a new page. i only made use of the create and not the new action from the comment controller @DeliceLydia

Comment (VICTOR-CHUKWU) [reply config/routes.rb]: Good job so far

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @Ginohmk , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by koachcode** [comment]
Thank you @VICTOR-CHUKWU  and @DeliceLydia

**Pull request merged**